Cargo.toml

```diff
@@ -21,12 +21,12 @@ exclude = [
 
 [features]
 default = ["tokio-runtime"]
-tokio-runtime = ["tokio/dns", "tokio/macros", "tokio/rt-core", "tokio/tcp", "tokio/rt-threaded", "tokio/time", "reqwest", "serde_bytes"]
-async-std-runtime = ["async-std", "async-std/attributes"]
+tokio-runtime = ["tokio/macros", "tokio/net", "tokio/rt", "tokio/time", "reqwest", "serde_bytes"]
+async-std-runtime = ["async-std", "async-std/attributes", "async-std-resolver", "tokio-util/compat"]
 sync = ["async-std-runtime"]
 
 [dependencies]
-async-trait = "0.1.24"
+async-trait = "0.1.42"
```

Comment:
In addition to updating tokio, I also updated all crates that use tokio under the hood, as well as async-trait